css/gallery.css

```diff
@@ -51,7 +51,156 @@
 
 .gallery__items :nth-child(2n) {
   margin-right: 0;
+}     
+.wiggle {
+  display:block;
+  margin:0 auto;
+  width:150px;
+  height:150px;
+  box-shadow:8px 8px 1px 0 rgba(0,0,0,.15);
 }
+ 
+.wiggle:hover {
+  -webkit-animation:none;
+}
+ 
+.short-animate {
+  -webkit-transition:.5s ease-in-out;
+  -moz-transition:.5s ease-in-out;
+  -ms-transition:.5s ease-in-out;
+  -o-transition:.5s ease-in-out;
+  transition:.5s ease-in-out;
+}
+ 
+.long-animate {
+  -webkit-transition: .5s .5s ease-in-out;
+  -moz-transition: .5s .5s ease-in-out;
+  -ms-transition: .5s .5s ease-in-out;
+  -o-transition:.5s .5s ease-in-out;
+  transition:.5s .5s ease-in-out;
+}
+ 
+html,body {
+  height:100%;
+  min-height:100%;
+}
+ 
+.lightbox {
+  position:fixed;
+  top:-100%;
+  bottom:100%;
+  left:0;
+  right:0;
+  background:#003c8f;
+  z-index:501;
+  opacity:0;
+}
+ 
+.lightbox img {
```

Comment:
I think you should assign a class to the img tag.

Reply:
Tal vez pero se le asigna esa clase para que solo afecta a las imàgenes en el lightbox.

Reply:
To follow the BEM standard it would be good to implement that suggestion, even when functonally its the same, it wold be cleanner and more flexible.

==========================
Para seguir el standard de BEM seria bueno implementar esa sugerencia (la de hanna), aun cuando funcionalmente no hace diferencia ahora, es un poco mas limpio y flexible a futuro.

te sugiero la clase `lightbox__image`

Reply:
PD llegue tarde, pero bueno, se puede tener en cuanta para la proxima XD